packages/experiments/package.json

```diff
@@ -68,6 +68,7 @@
     "@uifabric/theme-samples": "^7.0.22",
     "@uifabric/utilities": "^7.12.1",
     "@uifabric/variants": "^7.1.12",
+    "core-js": "3",
```

Comment:
We shouldn't add a dependency on core-js in this package. If you explain why you needed it, I can probably help find a workable alternative.

Reply:
whoops that should not be there. I'll remove it and update the PR

Reply:
Thanks! You'll need to update yarn.lock too.

Reply:
Just updated. Thanks!